provider/ec2/provider.go

```diff
@@ -129,13 +129,13 @@ func (p environProvider) metadataLookupParams(region string) (*simplestreams.Met
 	}, nil
 }
 
-const badKeys = `
+const badKeysFormat = `
 The provided credentials could not be validated and 
 may not be authorized to carry out the request.
 Ensure that your account is authorized to use the Amazon EC2 service and 
 that you are using the correct access keys. 
 These keys are obtained via the "Security Credentials"
-page in the AWS console.
+page in the AWS console. :%w
```

Comment:
I know this is existing, but I think we need to lose the "." and do `...page in the AWS console: %w`